### PR TITLE
RDBS-68 - fix delay in deleting expired documents

### PR DESCRIPTION
**Pull request by grisha-kotler**

### CODE REVIEW

**Comment by ppekrol** [comment]
Can we have a test for this? You can return number of removed/refreshed documents from `CleanupDocs` and use it in the test.

**Comment by grisha-kotler** [comment]
done

File: test/SlowTests/Issues/RDBS-68.cs [hunk 1]
```diff
@@ -0,0 +1,122 @@
...
+                    DeleteFrequencyInSec = 1000
+                };
+
+                await ExpirationHelper.SetupExpiration(store, Server.ServerStore, config);
+
+                var expiry = SystemTime.UtcNow.AddMinutes(-5);
+
+                var deleteList = new List<string>();
+
+                var totalInBatch = ExpiredDocumentsCleaner.BatchSize;
```
Comment (ppekrol) [inline test/SlowTests/Issues/RDBS-68.cs]: Either use smaller batch size or move this to stress

Comment (grisha-kotler) [reply test/SlowTests/Issues/RDBS-68.cs]: that's 4096 documents in total

Comment (ppekrol) [reply test/SlowTests/Issues/RDBS-68.cs]: Fixed.

**Pull request merged**